### ISSUE TITLE
Loading saved model with custom loss

### ISSUE THREAD

**Issue by cicobalico** [fchollet/keras#3457]
when I load a model that was saved in hdf5 that was using a custom loss function called `dice_coef_loss` I get the following exception
`Exception: Invalid objective: dice_coef_loss`
Even if I have defined the function in the current script it still raises this exeption.

Is this the normal behaviour? Do I have to install keras defining my custom loss in the source code?

**Comment by cicobalico** [comment]
Sorry guys, just noticed there's a lot of already open issues on the matter.
